spec/requests/swagger_docs/v6/full_assessment_spec.rb

```diff
@@ -108,26 +112,7 @@
                         cash_transactions: { "$ref" => components[:cash_transactions] },
                         outgoings: { type: :array },
                         irregular_incomes: { "$ref" => components[:irregular_income_payments] },
-                        employments: {
-                          type: :array,
-                          required: %i[name client_id payments],
-                          description: "One or more employment income details",
-                          items: {
-                            type: :object,
-                            description: "Employment income detail",
-                            properties: {
-                              name: {
-                                type: :string,
-                                description: "Identifying name for this employment - e.g. employer's name",
-                              },
-                              client_id: {
-                                type: :string,
-                                description: "Client supplied id to identify the employment",
-                              },
-                              payments: { "$ref" => components[:employment_payment_list] },
-                            },
-                          },
-                        },
+                        employments: { "$ref" => components[:employments] },
```

Comment:
@starswan With this change we will be adding additional attribute `receiving_only_statutory_sick_or_maternity_pay` and i think this change is correct  😄